src/algebra/big_operators/basic.lean

```diff
@@ -802,21 +802,25 @@ lemma mul_prod_diff_singleton [decidable_eq α] {s : finset α} {i : α} (h : i
   (f : α → β) : f i * (∏ x in s \ {i}, f x) = ∏ x in s, f x :=
 by { convert s.prod_inter_mul_prod_diff {i} f, simp [h] }
 
+/-- Partition a product by a setoid. -/
```

Comment:
I'm not sure this doc string is descriptive. It sounds like it applies to a function producing a partition.